library/src/main/java/org/apache/apex/malhar/lib/wal/WindowDataManager.java

```diff
@@ -126,36 +146,24 @@ public void teardown()
     }
 
     @Override
-    public void save(Object object, int operatorId, long windowId) throws IOException
+    public void save(Object object, long windowId) throws IOException
     {
     }
 
     @Override
-    public Object load(int operatorId, long windowId) throws IOException
+    public Object retrieve(long windowId) throws IOException
     {
       return null;
     }
 
     @Override
-    public void delete(int operatorId, long windowId) throws IOException
-    {
-    }
-
-    @Override
-    public void deleteUpTo(int operatorId, long windowId) throws IOException
```

Comment:
changed the method to be committed(long windowId). WindowDataManager is Evolving so this was ok.